.gitignore

```diff
@@ -31,3 +31,4 @@ npm-debug.log*
 /www
 /node_modules
 /android
+environment.ts
```

Comment:
@themuhammadattaurrehman remove this line, we cannot add environent.ts in gitignore
it should be the part of code